package/Cargo.toml

```diff
@@ -10,6 +10,7 @@ anyhow.workspace = true
 clap.workspace = true
 futures.workspace = true
 hex.workspace = true
+illumos-utils.workspace = true
```

Comment:
`omicron-package` is _nearly_ able to remove its dependency on `sled-agent`, but it still uses [`sled_agent::cleanup_networking_resources()`](https://github.com/oxidecomputer/omicron/blob/25f05bd648d42943a84f358a472f31578e88c505/package/src/bin/omicron-package.rs#L12). I think that function (and the handful of functions it calls) could be pulled out pretty easily, but I didn't do it as part of this PR (both because we don't need to for installinator and because it's not clear to me where to put it).

Reply:
I filed https://github.com/oxidecomputer/omicron/issues/2458to track